### ISSUE TITLE
Error in P03_BogieBearing

### ISSUE THREAD

**Issue by berendj** [esa-prl/ExoMy#8]
The four holes in the corners of this part have a bore of 4mm, while they are supposed to hold a 4mm bolt. The diameter of these holes should be 3.3mm

**Comment by mvoellmy** [comment]
While 3.3 mm is the right core hole for threading an M4, it would be too small here.
First, we recommend to "cut" the threads directly with the screws and don't use a thread-cutters. Screwing in a screw in a 3.3 mm whole would require too much force from our experience.
Second, a 4 mm hole in CAD will be much smaller in the print. Maybe around 3.7-.8 mm (never measured it). We found 4 mm to work well in terms of rigidity, reliability and ease of installation, so it should stay 4 mm.

**Comment by berendj** [comment]
Hello  Miro Voellmy,

 

Your first point is well taken, 3.3mm is for threading an M4. But I changed the design to 3.4mm and just did the threading with an M4 bolt without problems, but needed some force indeed.You wrote “Second, a 4 mm hole in CAD will be much smaller in the print. Maybe around 3.7-.8 mm (never measured it).”

As we say in Dutch “Meten is weten” as you may well know. A CAD designed 4mm hole IS 4mm, unless you never calibrated your 3D-printer.

I did and a 4mm bolt can be pushed through without force, apart from a little “elephant foot” at the bottom of the print.

 

I would appreciate you add a comment to the closed issue #8 to make others aware your choice and the consequence if they have calibrated their printer.

 

met vriendelijke groeten,

 

Bert Ruiterkamp

**Comment by mvoellmy** [comment]
Fair point regarding calibration. I assume you tuned your slicer using the Horizontal Expension parameter?
https://bradshacks.com/3d-printing-tolerancing/
I was not aware of this parameter and with that in mind I have to agree, that a smaller hole size would have been better.

I updated the wiki, adding a recommendation that people test the threads before printing all the parts.
https://github.com/esa-prl/ExoMy/wiki/3D-Printing

I will reopen the issue, but can not change it myself. As I'm not working at ESA anymore, I don't have access to Solidworks anymore nor the 3D printers (which should be calibrated ;) to test the fit. It would be good to change this in the future however.

**Comment by berendj** [comment]
Actually Cura has an even better feature. Horizontal Expansion enlarges or minimizes also the basic print size of your object. Cura has (in expert and all modes) a feature called Hole Horizontal Expansion. That leaves the overall object size unchanged and affects only the hole radius. See [this video](https://www.youtube.com/watch?v=UUelLZvDelU) for an explanation.

I had to reprint the bogies because the bolts for the servo's did not hold. I also hope the mast will hold on the chassis top. These holes also seem a bit wide.

**Comment by Penguin-fach** [comment]
Getting threads to hold reliably in PLA over time is problematic particularly with shock loading if you are going to drive over obstacles. I have found brass knurl nuts very easy to fit with a soldering iron and make an enormous difference to thread performance in all 3D printed parts. Has anyone tried this on ExoMy as yet?

Regards, Frazer

**Comment by mvoellmy** [comment]
Hey @Penguin-fach,
Agreed, brass inserts are amazing. I'm a big fan of them myself and an ExoMy with full brass inserts would definitely be great.

Currently we use brass inserts to fix the top to chassis. This is a part that we expect to be removed somewhat regularly. Thus the reusability you gain with the brass inserts should pay off.

On the other parts, we stuck with the direct cutting method to keep the complexity and assembly time of ExoMy low. So far we had threads which were too loose, but no failures in the threads occurred during operation.

We received an email of someone who wants to modify the STEP files of ExoMy to work with brass inserts, but I don't know what their current progress is.
